layers/meta-balena-5x-owa/recipes-support/hostapp-update-hooks/files/99-flash-bootloader

```diff
@@ -29,11 +25,8 @@ for i in $update_files; do
         update_size=$(ls -al /resin-boot/$current_update_file | awk '{print $5}')
         update_md5sum=$(md5sum /resin-boot/$current_update_file | awk '{print $1'})
 
-        # calculate number of bytes to skip when computing the checksum of the data we want to update (i.e. the data already written to $device)
-        let skip_bytes=$block_size*$seek_blocks
-
         # calculate md5sum of the data already written to $device, using $update_size bytes and skipping $skip_bytes from $device
-        existing_md5sum=$(dd if=$device skip=$skip_bytes bs=1 count=$update_size status=none | md5sum | awk '{print $1}')
```

Comment:
so u-boot gets written right at the the very start of this hw partition?

Reply:
No, bootloader is written in 32k offset, which you can see in line 41 of the original file, but...
you'r comment is quite legit, as the md5sum checking is the condition to update the bootloader, so, let me fix it and udpate it in a single PR as you said, thanks!

Reply:
Moved to this [PR](https://github.com/balena-os/balena-owa5x/pull/63)

Thanks